src/client/src/system/dimensions.jsx

```diff
@@ -0,0 +1,140 @@
+/*eslint-disable */
```

Comment:
I think we need to add the licencing information as per the initial projects licencing agreement. Additionally can we write a note as to why we've forked it locally.